packages/agoric-cli/lib/start.js

```diff
@@ -459,8 +462,11 @@ export default async function startMain(progname, rawArgs, powers, opts) {
         );
         // eslint-disable-next-line no-await-in-loop
         exitStatus = await capret[0];
-        if (!exitStatus && !capret[1].includes('code: 0')) {
-          exitStatus = 2;
+        if (!exitStatus) {
+          const ret = JSON.parse(capret[1]);
```

Comment:
This can throw and the error messages are opaque. Consider using a utility method that rethrows an informative error.

Reply:
Okay, I'll do that in the next batch of changes.